### ISSUE TITLE
play terminal failed command sound on navigation to command in accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#209580]
@jooyoungseo, I was thinking about it. Would it be useful if we played the terminal failed command sound as you navigate in the accessible view using `alt+up/down arrow` when it lands on a command with a non-zero exit code?

That way, you could quickly scan for the command with output you're interested in?

**Comment by jooyoungseo** [comment]
@meganrogge Wonderful idea! I like it. I have another suggestion related to Alt+Up/DownArrow command in terminal. Currently, screen reader does not anounce the focused line of the command when moving around with Alt+Up/DownArrows. Could we consider adding aria alert?

**Comment by meganrogge** [comment]
@jooyoungseo do you mean when in the terminal or in the terminal accessible view?

**Comment by jooyoungseo** [comment]
@meganrogge Even in terminal a11y view, alt+Up/DownArrow does not announce the currently focused command. Tested with JAWS.

**Comment by meganrogge** [comment]
That works for me when using VoiceOver, can you try with NVDA?

**Comment by meganrogge** [comment]
1. run some commands in the terminal, some of which fail (non zero exit code)
2. in screen reader mode, open the accessible view, `alt+f2`
3. use `alt+up/down arrow` to navigate 
4. verify that you hear the terminal command failed sound when the cursor lands on a line with a command that failed
